backend/src/main/java/com/htecgroup/SkyNest/service/impl/UserServiceImpl.java

```diff
@@ -51,4 +51,6 @@ public UserDto findUserByEmail(String email) {
 
     return modelMapper.map(userEntity, UserDto.class);
   }
+
```

Comment:
Try to format or check diffs before pushing, so we don't get changes like this :) .

Reply:
That's good suggestion, I'll check it from now.